weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx

```diff
@@ -85,6 +85,17 @@ export const CenterEntityBrowserInner: React.FC<
     props.entityName
   );
 
+  moment.updateLocale('en', {
+    calendar: {
+      sameDay: '[Today]',
+      nextDay: '[Tomorrow]',
+      nextWeek: 'dddd',
+      lastDay: '[Yesterday]',
+      lastWeek: '[Last] dddd',
+      sameElse: 'YYYY-MM-DD',
```

Comment:
This change removes the time suffix (e.g. " at 1:05 PM") in addition to changing the date format, was that intentional?

Reply:
no, that wasn't intentional. Pushed a change to add it.

Reply:
LGTM